build.go

```diff
@@ -29,11 +29,11 @@ import (
 
 var (
 	// buildContainer is a docker container used to build go binaries
-	buildContainer = "golang:1.14.3"
+	buildContainer = "golang:1.12.0"
 
 	// golangciVersion is the version of golangci-lint to use for linting
 	// https://github.com/golangci/golangci-lint/releases
-	golangciVersion = "v1.27.0"
+	golangciVersion = "v1.15.0"
```

Comment:
it's not clear to me if this is walking back a specific PR, but I think using a later golangciVersion isn't a problem here.

Reply:
Reverting golang version changes from https://github.com/gravitational/satellite/pull/214. I think specific golangci versions are only compatible with specific golang versions. Will just revert both for now.